migrations/913-websites-lookup-perms.sql

```diff
@@ -0,0 +1 @@
+UPDATE groups SET rules=CONCAT(rules, ',WebsiteLookup:View') WHERE name IN ('Staff', 'Support Staff', 'Carriers and Operators');
```

Comment:
Except the pattern is `<Noun>`:`<Verb>`. I don't have a problem with a few extra rules.